packages/ui-library/src/components/forms/text-input/index.css.ts

```diff
@@ -174,62 +175,62 @@ export const { tokenizedLight: textInputLight, tokenizedDark: textInputDark } =
       display: flex;
       justify-content: space-between;
       align-items: center;
-      border: ${Input.Default.Rest.width} ${Input.Default.Rest.style} ${Input.Default.Rest.color};
-      border-radius: ${InputBorderRadius};
+      border: ${inputfield.container.border.default.rest.width} ${inputfield.container.border.default.rest.style} ${inputfield.container.border.default.rest.color};
+      border-radius: ${inputfield.container.borderradius};
       box-sizing: border-box;
 
       .blr-input-icon {
         position: relative;
-        color: ${Placeholder.Default.Rest};
+        color: ${inputfield.placeholder.textcolor.default.rest};
```

Comment:
A similar thing to what's happening on Toggle Switch is happening here. It looks like the icon colour is different to what's in Figma. Unless the design has changed

Reply:
same as on toggle switch, the difference was expected and correct